packages/terra-application-name/src/ApplicationHeaderName.jsx

```diff
@@ -22,15 +24,17 @@ const ApplicationHeaderName = ({
   title,
   ...customProps
 }) => {
-  const ApplicationHeaderNameClassNames = cx([
+  const theme = React.useContext(ThemeContext);
+  const ApplicationHeaderNameClassNames = classNames(cx(
     'application-header-name',
-    customProps.className,
-  ]);
+    theme.className,
+  ),
+  customProps.className);
 
   return (
     <div {...customProps} className={ApplicationHeaderNameClassNames}>
-      {accessory && <div className={cx('accessory')}>{accessory}</div>}
-      {title && <div className={cx('title')}>{title}</div>}
+      {accessory && <div className={classNames(cx('accessory', theme.className))}>{accessory}</div>}
+      {title && <div className={classNames(cx('title', theme.className))}>{title}</div>}
```

Comment:
The theme name will get defined by `ApplicationHeaderNameClassNames` so theme.classname does no need to be specified on accessory or title

Reply:
Removed 4f672fe